addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1355,18 +1355,21 @@ export class PosStore extends Reactive {
         const baseUrl = this.session._base_url;
         return order.export_for_printing(baseUrl, headerData);
     }
-    async printReceipt() {
+    async printReceipt(order = this.get_order()) {
+        const nbrPrint = order.nb_print;
+        await this.data.write("pos.order", [order.id], { nb_print: nbrPrint + 1 });
```

Comment:
Shouldn't we write the value after it successfully printed?

Reply:
Yep